src/components/AdoptMeForm.js

```diff
@@ -54,72 +79,78 @@ const AdoptMeForm = props => {
     )
   })
 
-    if(props.formReveal === true)  {
-      return(
+  if (props.formReveal === true) {
+    if (submitted === false) {
```

Comment:
These if statements can be combined into one if you want, if I am grasping their implementation correctly.